runtime/src/serde_snapshot.rs

```diff
@@ -642,7 +646,11 @@ where
         genesis_config,
     );
 
-    accounts_db.maybe_add_filler_accounts(&genesis_config.epoch_schedule, &genesis_config.rent);
+    accounts_db.maybe_add_filler_accounts(
+        &genesis_config.epoch_schedule,
+        &genesis_config.rent,
+        genesis_config.epoch_schedule.get_epoch(a_slot),
```

Comment:
Instead of `a_slot`, can you use `snapshot_slot` from line 511?

Reply:
great idea
done